src/mumble/mumble_de.ts

```diff
@@ -1551,7 +1553,7 @@ Mumble unterstützt das Herabsetzen der Lautstärke anderer Anwendungen während
     </message>
     <message>
         <source>This shows the peak power of the speakers in the last frame (20 ms). Unless you are using a multi-channel sampling method (such as ASIO) with speaker channels configured, this will be 0. If you have such a setup configured, and this still shows 0 while you&apos;re playing audio from other programs, your setup is not working.</source>
-        <oldsource>This shows the peak power in the last frame (20 ms) of the speakers. Unless you are using a multi-channel sampling method (such as ASIO) with speaker channels configured, this will be 0. If you have such a setup configured, and this still shows 0 while you're playing audio from other programs, your setup is not working.</oldsource>
+        <oldsource>This shows the peak power in the last frame (20 ms) of the speakers. Unless you are using a multi-channel sampling method (such as ASIO) with speaker channels configured, this will be 0. If you have such a setup configured, and this still shows 0 while you&apos;re playing audio from other programs, your setup is not working.</oldsource>
```

Comment:
What is the `oldsource` tag even being used for? :thinking:

Reply:
Looks like Weblate does not show it.

It's additional data that I can imagine being useful. If the translation string changed, it could be helpful to the translator to see the change that was made. It may also be useful in other cases.

Anyway, not primary/critical-path workflow data.

Reply:
I do not know if or how Qt Linguist makes use of it.